README.md

```diff
@@ -3,12 +3,13 @@
 Container images for [Samba](https://samba.org) services.
 
 ### Our images:
-* Are [OCI](https://opencontainers.org/) formatted container images
-* Provide application-like high-level entrypoint scripts
+* Are [OCI](https://opencontainers.org/) formatted container images.
+* Provide application-like high-level entrypoint scripts.
```

Comment:
These bullet points are sentence fragments. They do not need punctuation. Adding punctuation to only some items and not all of them makes the content inconsistent. Please remove the periods that this PR is adding

Reply:
done